### PR TITLE
OAuth: skip authorization email if active userToken

### PR DESCRIPTION
**Pull request by znarf**

### CODE REVIEW

File: server/lib/oauth/model.ts [hunk 1]
```diff
@@ -159,16 +159,22 @@ const model: OauthModel = {
...
-      CollectiveId: user.CollectiveId,
-      data: {
-        application: application.publicInfo,
-        collective: collective.minimal,
-        scope,
-      },
+    // Only send the email if there is no active user token right now
+    const userToken = await models.UserToken.findOne({
+      where: { ApplicationId: application.id, UserId: user.id },
     });
```
Comment (Betree) [inline server/lib/oauth/model.ts:165]: A `model.exists` would be nice for those cases (https://github.com/sequelize/sequelize/issues/10187)

**Review by Betree — APPROVED** [review]
Looks good. We'll probably want to send another kind of email when the permissions change

**Comment by znarf** [comment]
@Betree asking for new scopes definitely deserve another email.

**Comment by znarf** [comment]
@Betree asking for extra review as I implemented the scope check.

**Review by Betree — APPROVED** [review]
:+1:

**Pull request merged**